Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:2.7
+WORKDIR /code
+ADD . /code
+ENV LINTREVIEW_SETTINGS /code/settings.sample.py
+RUN apt-get update && apt-get install -y php-pear ruby npm && \
```

Comment:
Makes sense. Nothing is being removed though, does the `autoremove` actually clean up any packages all this does is install stuff?
Reducing bloat is awesome, but caching different stages of the build is also awesome. 
Would splitting each package manager up make sense?

Something like this:
```
RUN apt-get update && apt-get -y dist-upgrade && apt-get install -y php-pear ruby npm && apt-get -y clean
RUN ln -s /usr/bin/nodejs /usr/bin/node
RUN pear install PHP_CodeSniffer
RUN gem install bundler && bundle install --system
RUN npm install -y csslint jshint
RUN pip install -r requirements.txt && pip install .
```

It's been a little while since I've used docker, so I don't know what best practise is.